pkg/cloudprovider/controller/nodecontroller.go

```diff
@@ -34,6 +34,24 @@ import (
 	"github.com/golang/glog"
 )
 
+const (
+	// The constant is used if sync_nodes_status=False. NodeController will not proactively
+	// sync node status in this case, but will monitor node status updated from kubelet. If
+	// it doesn't receive update for this amount of time, it will start posting node NotReady
+	// condition. The amount of time when NodeController start evicting pods is controlled
+	// via flag 'pod_eviction_timeout'. Note: be cautious when changing nodeMonitorGracePeriod,
```

Comment:
Without a history of node conditions, we cant count the age of last node Ready I think.  So I think you are right.  I'll update the comment.